src/inmanta/logging.py

```diff
@@ -593,35 +574,7 @@ def apply_options(self, options: Options) -> None:
         if self._options_applied:
             raise Exception("Options can only be applied once to a handler.")
 
-        logging_config_file: Optional[str] = self._get_path_to_logging_config_file(options)
-        if logging_config_file:
-            self._apply_logging_config_from_file(logging_config_file)
-        else:
-            self._apply_logging_config_from_options(options)
```

Comment:
We should set the `_options_applied` flag at the end of this method. Otherwise we might get into an inconsistent state if an exception occurs.